src/parameter_input.cpp

```diff
@@ -397,7 +397,7 @@ void ParameterInput::ModifyFromCmdline(int argc, char *argv[]) {
 
   for (int i = 1; i < argc; i++) {
     input_text = argv[i];
-    std::size_t slash_posn = input_text.find_first_of("/"); // find "/" character
+    std::size_t slash_posn = input_text.rfind("/");         // find last "/" character
     std::size_t equal_posn = input_text.find_first_of("="); // find "=" character
```

Comment:
This is not something that you introduced, but there should be a check to see whether the equals position is greater than slash_posn+1.   Or the slash search should be in the substring [:equal_posn].  Otherwise line 408 will crash:
````C++
    name = input_text.substr(slash_posn + 1, (equal_posn - slash_posn - 1));
````
It is possible that this is not a case that we will encounter.  If so, then feel free to ignore this statement.

Reply:
Good catch. Not completely unlikely that a user is going to use a `/` in the parameter file within a value.
Updated result now looks like
```
example/calculate_pi/pi-example  -i ../example/calculate_pi/parthinput.example parthenon/mesh/nx1=64/3
### PARTHENON ERROR
  Message:     '/' used as value (rhs of =) when modifying parthenon/mesh/nx1=64/3.Please update value of change logic in ModifyFromCmdline function.
  File:        /home/pgrete/src/parthenon/src/parameter_input.cpp
  Line number: 407
```